lib/src/icons/Carat-Down.scss

```diff
@@ -0,0 +1,11 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+.nx-icon--carat-down {
+  color: #000000;
```

Comment:
I'd argue that the color should be left up to the usage, not built into the icon.  i.e. this should be specified in the NxDropdown styles.

Reply:
> I'd argue that the color should be left up to the usage, not built into the icon. i.e. this should be specified in the NxDropdown styles.

Makes sense: https://github.com/sonatype/sonatype-react-shared-components/pull/55/commits/5c6c68604acbee2b127d74adafd508f40ee0d035